src/Redux/fireRequest.tsx

```diff
@@ -131,6 +131,11 @@ export const fireRequest = (
             return;
           }
 
+          // 409 is for conflict on access to an asset
+          if (error.response.status === 409 && key === "operateAsset") {
+            return error.response;
+          }
+
```

Comment:
add this to onResponse of the request options after migrating to useQuery/request and remove it from firerequest